lib/mayaUsd/undo/OpUndoItemMuting.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <mayaUsd/undo/OpUndoItemMuting.h>
+
```

Comment:
All code in the C++ file is not new but moved from the header file, now that there is more code, including a base class, inlining it makes less sense.